app/src/components/widgets/tables/productStatus/itemLink/itemLink.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import track from 'react-tracking';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { NameLink } from 'pages/inside/common/nameLink';
+import { LAUNCHES_PAGE_EVENTS } from 'components/main/analytics/events';
+import { formatItemName } from 'controllers/testItem';
+import styles from './itemLink.scss';
+
+const cx = classNames.bind(styles);
+
+@track()
+export class ItemLink extends Component {
+  static propTypes = {
+    value: PropTypes.object,
+    customProps: PropTypes.object,
+    tracking: PropTypes.shape({
+      trackEvent: PropTypes.func,
+      getTrackingData: PropTypes.func,
+    }).isRequired,
+  };
+
+  static defaultProps = {
+    value: {},
+    customProps: {
+      ownLinkParams: {},
+    },
+  };
+
+  render() {
+    const { value, tracking, customProps } = this.props;
+    return (
+      <div className={cx('main-info')}>
+        <NameLink
```

Comment:
I think not needed to create the separate component here. Just use `NameLink` component with necessary props inside the `NameColumn`